docs/user_guide/spin_system/spin_system.rst

```diff
@@ -4,10 +4,56 @@
 Spin System
 ===========
 
-At the heart of any mrsimulator calculation is the definition of a :ref:`spin_sys_api`
-object describing the sites and couplings within a spin system. Each :ref:`simulator_api` object
-holds a list of :ref:`spin_sys_api` objects which will be used to calculate frequency
-contributions.
+At the heart of any ``mrsimulator`` calculation is the definition of
```

Comment:
mrsimulator should either be in plain text or bolded, but not code text

Reply:
I think bold should be the default for **mrsimulator**

Reply:
and also for Object names.